lib/puma/rack/builder.rb

```diff
@@ -1,8 +1,5 @@
 # frozen_string_literal: true
 
-module Puma
```

Comment:
Whoops, when did that get added?

Reply:
https://github.com/puma/puma/commit/5fdf337790a62bdcb8a7db3379cba4007426ebec#diff-e472dc9fd53aac8bac37c1308caeaa854989c40f4d0c21e07b54dfca06887db1